### PR TITLE
FSMA-83 Fix issue to map gl to specific gl for disburse to savings ch…

### PR DESCRIPTION
**Pull request by Deepika1095**
…arge

https://fiterio.atlassian.net/browse/FSMA-83

## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by hagafrank — DISMISSED** [review]

File: fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java [hunk 1]
```diff
@@ -125,6 +125,7 @@ public LoanDTO populateLoanDtoFromMap(final Map<String, Object> accountingBridge
             final BigDecimal overPayments = (BigDecimal) map.get("overPaymentPortion");
             final boolean reversed = (Boolean) map.get("reversed");
             final Long paymentTypeId = (Long) map.get("paymentTypeId");
+            boolean isDisburseToSavings = (Boolean) map.get("isDisburseToSavings");
```
Comment (oluexpert99) [inline fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java]: use get or default , or ell you might get a NULL pointer exception

File: fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java [hunk 2]
```diff
@@ -125,6 +125,7 @@ public LoanDTO populateLoanDtoFromMap(final Map<String, Object> accountingBridge
             final BigDecimal overPayments = (BigDecimal) map.get("overPaymentPortion");
             final boolean reversed = (Boolean) map.get("reversed");
```
Comment (oluexpert99) [inline fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java:126]: use get or default

File: fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java (hunk 1 above)
Comment (Deepika1095) [reply fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java]: I added that particular field from both places from where we are using it. but let me update it.

Comment (oluexpert99) [reply fineract-provider/src/main/java/org/apache/fineract/accounting/journalentry/service/AccountingProcessorHelper.java]: final boolean reversed = (Boolean) map.getOrDefault("reversed", false);
boolean isDisburseToSavings = (boolean) map.getOrDefault("isDisburseToSavings", false);

**Review by oluexpert99 — APPROVED** [review]
LGTM

**Review by hagafrank — APPROVED** [review]

**Pull request merged**